qiime2/plugin/tests/test_plugin.py

```diff
@@ -151,7 +151,7 @@ def test_pipelines(self):
     # TODO test registration of directory formats.
 
     def test_types(self):
```

Comment:
Please rename:

```suggestion
    def test_type_fragments(self):
```

Also, please add tests for `plugin.types` (if you need some ideas on how to go about this please let me know - for now, I would suggest trying to implement a test similar to the one described on L153).